### PR TITLE
cloudscale migrated to cloudscale_ch.cloud

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
Plugins migrated to vendor repository https://github.com/cloudscale-ch/ansible-collection-cloudscale. 
Routing in ansible/ansible will be adjusted later.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
cloudscale

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by felixfontein** [comment]
@resmo can you rebase? I guess I created a conflict with my deprecation updating PR (probably something in cloudscale_server.py).

File: tests/sanity/ignore-2.10.txt [hunk 1]
```diff
@@ -194,12 +194,7 @@ plugins/modules/cloud/centurylink/clc_server_snapshot.py validate-modules:doc-mi
 plugins/modules/cloud/centurylink/clc_server_snapshot.py validate-modules:implied-parameter-type-mismatch
 plugins/modules/cloud/centurylink/clc_server_snapshot.py validate-modules:parameter-list-no-elements
 plugins/modules/cloud/centurylink/clc_server_snapshot.py validate-modules:parameter-type-not-in-doc
-plugins/modules/cloud/cloudscale/cloudscale_floating_ip.py validate-modules:doc-required-mismatch
-plugins/modules/cloud/cloudscale/cloudscale_server.py validate-modules:doc-required-mismatch
-plugins/modules/cloud/cloudscale/cloudscale_server.py validate-modules:parameter-list-no-elements
-plugins/modules/cloud/cloudscale/cloudscale_server_group.py validate-modules:doc-required-mismatch
-plugins/modules/cloud/cloudscale/cloudscale_volume.py validate-modules:doc-required-mismatch
-plugins/modules/cloud/cloudscale/cloudscale_volume.py validate-modules:parameter-list-no-elements
+plugins/modules/cloud/digital_ocean/digital_ocean.py validate-modules:deprecation-mismatch
```
Comment (felixfontein) [inline tests/sanity/ignore-2.10.txt:197]: This line isn't needed.

**Comment by resmo** [comment]
Thanks!

**Comment by felixfontein** [comment]
@resmo thanks for the PR! :)

**Pull request merged**